src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -498,7 +484,7 @@ public String getByteLengthFunction(Interface service, Method method, TypeRef ty
     switch (typeRef.getKind()) {
       case TYPE_MESSAGE:
         return "gax.createByteLengthFunction(grpcClients."
-            + getStubNameFor(service, method)
+            + getApiMethodView(service, method).stubName()
```

Comment:
This was a typo -- it turned out to be `grpcClientVariableName()`. Are there good name for that?